packages/nextjs/services/database/bg-imports/20250423_location_socials.ts

```diff
@@ -0,0 +1,101 @@
+import { db } from "../config/postgresClient";
+import { lower, users as schemaUsers } from "../config/schema";
+import * as dotenv from "dotenv";
+import { eq } from "drizzle-orm";
+import * as path from "path";
+
+dotenv.config({ path: path.resolve(__dirname, "../../../.env.development") });
+
+const BG_USERS_ENDPOINT = `${process.env.NEXT_PUBLIC_BG_BACKEND}/builders`;
+
+export type BGBuildersApiResponse = {
+  id: string;
+  socialLinks?: {
+    twitter?: string;
+    github?: string;
+    telegram?: string;
+    discord?: string;
+    instagram?: string;
+    email?: string;
+  };
+  role?: string;
+  function?: string;
+  creationTimestamp?: number;
+  builds?: { submittedTimestamp: number; id: string }[];
+  ens?: string;
+  status?: { text?: string; timestamp?: number };
+  reachedOut?: boolean;
+  scholarship?: boolean;
+  batch?: { number?: string; status?: string };
+  location?: string;
+  builderCohort?: { name?: string; id?: string; url?: string }[];
+  stream?: {
+    lastContract?: number;
+    cap?: string;
+    lastIndexedBlock?: number;
+    balance?: string;
+    streamAddress?: string;
+    frequency?: number;
+  };
+  [key: string]: any;
+}[];
+
+export async function importLocationAndSocials() {
```

Comment:
nit: why to export it?

Reply:
Ohh we had it in batch data import script too : 

https://github.com/BuidlGuidl/SpeedRunEthereum-v2/blob/f87a9ac814b56bfc3b99680d170ef21c76ab4292/packages/nextjs/services/database/bg-imports/20250416_batch_data.ts#L32
 
 But happy to remove it if we feel right that way 🙌

Reply:
removed at https://github.com/BuidlGuidl/SpeedRunEthereum-v2/pull/148/commits/526711a4dc0cfc8e39750967bfab6d38912a4ceb